firebase-appdistribution-api/src/main/java/com/google/firebase/appdistribution/AppDistributionRelease.java

```diff
@@ -0,0 +1,41 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.appdistribution;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+/**
+ * The release information returned by {@link FirebaseAppDistribution#checkForNewRelease} when a new
+ * version is available for the signed in tester.
+ */
+public interface AppDistributionRelease {
```

Comment:
That's not true. All methods on interfaces are (implicitly) public, but the interface itself can be package private. It needs to be explicitly public, so that code from other packages can use it.

Reply:
You're right, my mistake.